### ISSUE TITLE
Requests supports urllib3 v1.25.7 ?

### ISSUE THREAD

**Issue by Rogacz** [psf/requests#5272]
Hi Guys,
based on [https://2.python-requests.org/en/master/community/updates/#id1](https://2.python-requests.org/en/master/community/updates/#id1) requests should support urllib3 v1.25.2 and greater(?)
The `__init__.py` file doesn't reflect that.

## Expected Result

Support for urllib3 v1.25.7 

## Actual Result

/usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25.7) or chardet (2.2.1) doesn't match a supported version!

## Reproduction Steps

```python
import requests

```

## System Information

```
python -m requests.help
/usr/lib/python2.7/site-packages/requests/__init__.py:91: RequestsDependencyWarning: urllib3 (1.25.7) or chardet (2.2.1) doesn't match a supported version!
  RequestsDependencyWarning)
{
  "chardet": {
    "version": "2.2.1"
  },
  "cryptography": {
    "version": "2.8"
  },
  "idna": {
    "version": "2.7"
  },
  "implementation": {
    "name": "CPython",
    "version": "2.7.5"
  },
  "platform": {
    "release": "3.10.0-1062.4.3.el7.x86_64",
    "system": "Linux"
  },
  "pyOpenSSL": {
    "openssl_version": "100020bf",
    "version": "0.13.1"
  },
  "requests": {
    "version": "2.22.0"
  },
  "system_ssl": {
    "version": "100020bf"
  },
  "urllib3": {
    "version": "1.25.7"
  },
  "using_pyopenssl": true
}
```

**Comment by sethmlarson** [comment]
It's your chardet version that isn't valid, should be >=3.0.2.

Fix with `$ python -m pip install chardet>=3.0.2`

**Comment by Rogacz** [comment]
Thanks, pip list was showing chardet as 3.0.4 so that's why I thought it's fine. I did force reinstall it and now there is no error.
